containerbuddy/config.go

```diff
@@ -311,8 +312,15 @@ func initializeConfig(config *Config) (*Config, error) {
 			return nil, ifaceErr
 		}
 
-		if service.ipAddress, err = GetIP(interfaces); err != nil {
-			return nil, err
+		if service.IpAddress == "" {
+			if service.IpAddress, err = GetIP(interfaces); err != nil {
+				return nil, err
+			}
+		} else {
+			if err := net.ParseIP(service.IpAddress); err == nil {
```

Comment:
The test you've written exercises the parsing of the config file but doesn't exercise this new code. This line should read `if err := net.ParseIP(service.IpAddress); err != nil {` and this wasn't caught by the tests.